### DISCUSSION TITLE
Каков шанс запороть роутер?

### DISCUSSION THREAD

**Discussion by 8hrsk** [bol-van/zapret#982] [Q&A] [answered]
Я, сам по себе, дилетант в этой теме. Хочу накатить zapret на роутер TP-Link Archer C54, но терзают смутные сосмнения. Каков шанс убить роутер и есть ли он вообще? В случае, если есть, можно ли будет оживить его без танцев с бубном (a.k.a программатором)? 

Заранее спасибо за ответ!

**Comment by BubaMuba** [comment]
Уже писали в какой-то из тем, что flash при постоянной перезаписи долго не живет. Если бы достаточно было раз настроить и забыть, то ваш C54 подошел бы. Но так не получится. Будете обновлять ПО, подбирать конфиг, железка сдохнет. Ищите что-то с usb портом или слотом для SD карты, делайте на внешнем носителе ext-root, тогда будет жить долго.

**Reply by 8hrsk** [reply]
Может, есть тогда вариант развернуть запрет в Docker и использовать его в качестве прокси?
В данный момент переживаю не самую лучшую финансовую ситуацию и покупать роутер (при наличии рабочего) не очень хотелось бы.
Однако, у меня есть хост с белым IP и доменным именем. Провайдер везде один и тот же (в теории, сконфигурировать на хосте стратегии запрет будет не трудно)

**Reply by bol-van** [reply, marked as answer]
extroot, конечно, очень хорошо, Но openwrt специально создавался с учетом минимизации записи.
Там многое в /tmp хранится.
zapret тоже пишет немного. Главная нагрузка по записи - скачиваемые листы. Еще логи autohostlist, если включены. Остальное - мелочи. Если нет автолиста, нет загрузок листов, то остается лишь ресолвинг IP из user листов, выполняющийся раз в 2 дня по cron.
